src/ducks/transactions/Transactions.jsx

```diff
@@ -129,35 +126,60 @@ const TableTrNoDesktop = translate()(
                   getCategoryId(transaction)
                 )}`
               )}
+              onClick={() => selectTransaction(transaction)}
             >
               <CategoryIcon
                 category={getParentCategory(getCategoryId(transaction))}
               />
             </Img>
             <Bd className="u-mr-half u-ellipsis">
-              <ListItemText
-                primaryText={getLabel(transaction)}
-                secondaryText={
-                  !filteringOnAccount && getAccountLabel(transaction.account)
-                }
-              />
+              <div onClick={() => selectTransaction(transaction)}>
+                <ListItemText
+                  primaryText={getLabel(transaction)}
+                  secondaryText={
+                    !filteringOnAccount && getAccountLabel(transaction.account)
+                  }
+                />
+              </div>
             </Bd>
-            <Img style={{ flexBasis: '1rem' }}>
-              <TransactionActions
-                transaction={transaction}
-                {...props}
-                onlyDefault
-                onlyIcon
-              />
-            </Img>
-            <Img>
+            <Img onClick={() => selectTransaction(transaction)}>
               <Figure
                 total={transaction.amount}
                 currency={transaction.currency}
                 coloredPositive
                 signed
               />
             </Img>
+            <Img
+              style={{
+                position: 'relative',
+                flexBasis: '2rem',
+                flexShrink: '0',
+                flexGrow: '0',
+                textAlign: 'right',
+                marginLeft: '1rem',
+                paddingLeft: '0.4rem'
+              }}
+            >
+              <span
+                style={{
+                  position: 'absolute',
+                  height: '1.5rem',
+                  width: '0.0635rem',
+                  left: '-0.0635rem',
+                  top: '50%',
+                  transform: 'translateY(-50%)',
+                  backgroundColor: 'silver'
+                }}
```

Comment:
same here. why do we need such custom CSS ?

Reply:
what's this ?

Reply:
This border was on the compact version of the `<ButtonAction />` at the beginning, but I removed it and put it in banks because it doesn't make sense to have a left border on every button. Also, we need a border even if there is no button on the row in banks. So I decided this border was the app's concern.